test_utils/sample_responses/jobs.py

```diff
@@ -30,13 +30,13 @@
             'duplicate_postings': 5551
         },
         'ranking': {
-            'facet': 'company',
+            'facet': 'title',
             'rank_by': 'unique_postings',
             'limit': 2,
             'buckets': [
                 {
                     'median_posting_duration': 22,
-                    'name': 'Apple Inc.',
+                    'name': '15.0120',
```

Comment:
Any reason to change it to number? Is this how the API will be returning us the results?

Reply:
Previously we were using TITLE_NAME facet, and now we are using TITLE Facet, which returns ids of the job instead of the name of the jobs. 
`15.0120` is an ID.